.github/workflows/ci.yaml

```diff
@@ -3,25 +3,23 @@ name: myCI
 on:
   #workflow_dispatch: #manual start from Github UI or API
   push:
-    branches: master
-  #pull_request:
-    #branches: master
+    branches: [master, dev]
+  pull_request:
+    branches: [master, dev]
 
 jobs:
-  main_job:
-    runs-on: ubuntu-latest
-    permissions:      
-      contents: read  #permissions for publishing Docker images to Github Packages
-      packages: write #permissions for publishing Docker images to Github Packages
-          
+  test_job:
```

Comment:
make shorter semantic names
like `test` is enough no need to append `_job` etc.

Reply:
ok